codeforboston/maple
# Typesense collection backfill fix

# Summary

Adds a try catch when we're going through the collection documents and backfilling the updated collection with the new schema.

# Checklist


# Screenshots

_Add some screenshots highlighting your changes._

# Known issues

_If you've run against limitations or caveats, include them here. Include follow-up issues as well._

# Steps to test/reproduce

_For each feature or bug fix, create a step by step list for how a reviewer can test it out. E.g.:_

1. Go to Browse Testimony
2. Check if private users have the search hidden from the RefinementList
